src/maint/nrepl/impl/docs.clj

```diff
@@ -0,0 +1,146 @@
+(ns nrepl.impl.docs
```

Comment:
I guess you can add some docstring overview of the purpose of the ns. That's always useful for people exploring the code.

Reply:
Added in latest revision